ush/generate_FV3LAM_wflow.py

```diff
@@ -919,9 +823,10 @@ def setup_logging(logfile: str = "log.generate_FV3LAM_wflow", debug: bool = Fals
     fh.setLevel(logging.DEBUG)
     fh.setFormatter(formatter)
     logging.getLogger().addHandler(fh)
-    logging.debug(f"Finished setting up debug file logging in {logfile}")
+    logging.debug("Finished setting up debug file logging in {logfile}")
```

Comment:
This f-string removal looks incorrect to me.
```suggestion
    logging.debug(f"Finished setting up debug file logging in {logfile}")
```